docs/source/install/prerequisites.rst

```diff
@@ -205,30 +209,77 @@ Make sure that RabbitMQ is running with:
     If you still have trouble getting RabbitMQ to run, please refer to the detailed instructions provided on the `website of RabbitMQ itself for generic Unix systems <https://www.rabbitmq.com/install-generic-unix.html>`_.
 
+
 .. _details_wsl:
 
 Windows Subsystem for Linux (Ubuntu)
 ====================================
 
-The guide for Ubuntu above can be followed but there are a few things to note:
+The guide for Ubuntu above can generally be followed, but there are a few things to note:
 
-#. The `windows native RabbitMQ <https://www.rabbitmq.com/install-windows.html>`_
-   should be installed and started,
-   since the version in WSL Ubuntu 18.04 does not work properly.
+#. It may be neccessary to install and start the `Windows native RabbitMQ <https://www.rabbitmq.com/install-windows.html>`_.
 
 #. Linux services under WSL are not started automatically.
-   To start the PostgreSQL service, type the command below in the terminal::
+   To start the PostgreSQL and RabbitMQ-server services, type the command below in the terminal::
 
      sudo service postgresql start
+     sudo service rabbitmq-server start
+
+   .. tip::
+
+       These services may be run at startup *without* passing a password in the following way:
+
+       Create a ``.sh`` file with the lines above, but *without* ``sudo``.
+       Make the file executeable, i.e., type::
+
+         chmod +x /path/to/file.sh
+
+       Then type::
+
+         sudo visudo
+
+       And add the line::
+
+         <username> ALL=(root) NOPASSWD: /path/to/file.sh
+
+       Replacing ``<username>`` with your Ubuntu username.
+       This will make you able to run *only* this specific ``.sh`` file with ``root`` access, i.e., not passing a password, without lowering security on your system.
 
-#. There is a `known issue <https://github.com/Microsoft/WSL/issues/856>`_ in WSL Ubuntu 18.04 where the timezone is not
-   configured correctly out-of-the-box, which may cause problem for the database.
+#. There is a `known issue <https://github.com/Microsoft/WSL/issues/856>`_ in WSL Ubuntu 18.04 where the timezone is not configured correctly out-of-the-box, which may cause problem for the database.
    The following command can be used to re-configure the time zone::
 
      sudo dpkg-reconfigure tzdata
 
 #. The file open limit may need to be raised using ``ulimit -n 2048`` (default is 1024), when running tests.
    You can check the limit by using ``ulimit -n``.
 
+   .. hint:: This may need to be run every time the system starts up.
+
 It may be worth considering adding some of these commands to your ``~/.bashrc`` file, since some of these settings may reset upon reboot.
 
+.. hint::
+
+    Installing `Ubuntu <https://www.microsoft.com/en-gb/p/ubuntu/9nblggh4msv6?source=lp&activetab=pivot:overviewtab>`_ instead of the version specific applications, will let you have the latest LTS version.
+
+.. note::
```

Comment:
Is this is specific to WSL?
in any case, this should probably go in the developer wiki
https://github.com/aiidateam/aiida-core/wiki/Development-environment